app/validators/sections/section_validator.py

```diff
@@ -96,6 +96,15 @@ def validate_blocks(self, group_id):
 
             self.validate_question(block)
             self.validate_variants(block)
+            # Repeating blocks must be validated here instead of from ListCollectorValidator as the latter cannot do standard block validation
```

Comment:
Can you not do this code in list collector validator? something like:

```
for repeating_block in self.block.get("repeating_blocks", []):
    section_validator = SectionValidator(repeating_block, self.questionnaire_schema)
    block_validator = get_block_validator(
        repeating_block, self.questionnaire_schema
    )
    self.errors += block_validator.validate()

    section_validator.validate_question(repeating_block)
    section_validator.validate_variants(repeating_block)
```

Reply:
This causes a circular dependancy - it's also passing a block to the section parameter so feel like a misuse. 
I think moving RB validation to ListCollectorValidator could be done by moving lots of SectionValidator out in a generic validator helper, but feels out of scope here for the scale of changes it'd be.